pkg/api/gql_client.go

```diff
@@ -109,7 +110,22 @@ func (c *GQLClient) Do(query string, variables map[string]interface{}, response
 // to the GitHub GraphQL schema.
 // Provided input will be set as a variable named input.
 func (c *GQLClient) MutateWithContext(ctx context.Context, name string, m interface{}, variables map[string]interface{}) error {
-	return c.client.MutateNamed(ctx, name, m, variables)
+	err := c.client.MutateNamed(ctx, name, m, variables)
+	var gqlErrs graphql.Errors
+	if err != nil && errors.As(err, &gqlErrs) {
+		items := make([]GQLErrorItem, len(gqlErrs))
+		for i, e := range gqlErrs {
+			items[i] = GQLErrorItem{
+				Message:    e.Message,
+				Locations:  e.Locations,
+				Path:       e.Path,
+				Extensions: e.Extensions,
+				Type:       e.Type,
+			}
+		}
+		err = &GQLError{items}
```

Comment:
This change could easily be backwards-compatible if GQLError implemented `Unwrap() error` that returned the original `graphql.Errors` instance